sdk/keyvault/azure-security-keyvault-keys/test/perf/inc/azure/keyvault/keys/test/get_key.hpp

```diff
@@ -46,10 +46,10 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
       m_tenantId = m_options.GetMandatoryOption<std::string>("TenantId");
       m_clientId = m_options.GetMandatoryOption<std::string>("ClientId");
       m_secret = m_options.GetMandatoryOption<std::string>("Secret");
-      m_credentail = std::make_shared<Azure::Identity::ClientSecretCredential>(
+      m_credential = std::make_shared<Azure::Identity::ClientSecretCredential>(
           m_tenantId, m_clientId, m_secret);
       m_client
-          = std::make_unique<Azure::Security::KeyVault::Keys::KeyClient>(m_vaultUrl, m_credentail);
+          = std::make_unique<Azure::Security::KeyVault::Keys::KeyClient>(m_vaultUrl, m_credential);
```

Comment:
XD, I wonder how @ahsonkhan  missed this typo =P

Reply:
I didn't review many of the keyault PRs :)